src/test/java/com/github/webicitybrowser/spec/fetch/FetchEngineTest.java

```diff
@@ -47,7 +53,22 @@ public void testFetchWithMockData() {
 		FetchConsumeBodyAction consumeBodyAction = Mockito.mock(FetchConsumeBodyAction.class);
 		Mockito.doNothing().when(consumeBodyAction).execute(Mockito.any(), Mockito.anyBoolean(), Mockito.any());
 
-		FetchEngine fetchEngine = new FetchEngineImp(mockConnectionPool());
+		FetchEngine fetchEngine = new FetchEngineImp(new FetchConnectionPool() {
```

Comment:
Is this an actual instance of the HTTP service, retrieving data from Google? Tests should not do IO.
Also, if so, how is this passing? I wouldn't expect whatever Google returns not to match the dummy data.

Lastly, can you add a DisplayName annotation to this test?

Reply:
I dont know what is retrieved from Google but my purposee here was to only get SUCCESS status code from HTTP fetch. This test is yet to reafctor because i wanted to execute it somewhere.